### PR TITLE
docs(ecs): Update docs to note Fargate support

### PR DESCRIPTION
**Pull request by clareliguori**
Fargate support has been in the ECS provider since [version 1.10](https://github.com/spinnaker/clouddriver/pull/2909), but I'm just now getting around to updating the docs

### CODE REVIEW

File: setup/install/providers/aws/aws-ecs.md [hunk 1]
```diff
@@ -14,11 +14,14 @@ account](https://aws.amazon.com/account/){:target="\_blank"}.
 ## Prerequisites
 
 ### ECS cluster
```
Comment (dorbin) [inline setup/install/providers/aws/aws-ecs.md]: "Amazon ECS"? :smiley:

File: setup/install/providers/aws/aws-ecs.md [hunk 2]
```diff
@@ -14,11 +14,14 @@ account](https://aws.amazon.com/account/){:target="\_blank"}.
...
-You need to [create an ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create_cluster.html){:target="\_blank"}
-and have enough capacity in it to deploy your containers.  
+
+You need to [create an ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create_cluster.html){:target="\_blank"}. If using the 'EC2' launch type, this cluster must have enough EC2 instance capacity in it to deploy your containers.  If using the 'Fargate' launch type, there is no need to add any capacity to this cluster.
```
Comment (dorbin) [inline setup/install/providers/aws/aws-ecs.md]: s/there is no need to/you don't need to/

File: setup/install/providers/aws/aws-ecs.md [hunk 3]
```diff
@@ -14,11 +14,14 @@ account](https://aws.amazon.com/account/){:target="\_blank"}.
...
 
 ### Networking
-As Elastic Network Interfaces (ENIs) are not yet supported in Spinnaker, you do not need to setup any further networking.  The cluster's networking configuration will be passed from your cluster instances to your containers.   
+
+If using the 'awsvpc' networking mode (required for the 'Fargate' launch type), you need a VPC with at least one subnet group and security group visible in Spinnaker.
+
+If using other networking modes like 'bridge', you do not need to setup any further networking.  The cluster's networking configuration will be passed from your cluster's EC2 instances to your containers.
```
Comment (dorbin) [inline setup/install/providers/aws/aws-ecs.md]: s/do not/don't/

**Review by dorbin — APPROVED** [review]

File: setup/install/providers/aws/aws-ecs.md (hunk 1 above)
Comment (clareliguori) [reply setup/install/providers/aws/aws-ecs.md]: lol will fix in other PR :)

**Review by dorbin — APPROVED** [review]

**Pull request merged**